src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -101,7 +97,7 @@ export const NavBar: React.FC<NavBarPropsType> = ({
 
   const {
     location: { pathname },
-    push: openUrlUsingRouter,
+    // push: openUrlUsingRouter,
```

Comment:
let's remove it

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1492/commits/fd5e29166eac851e32064ef6881374403dd1132d